extension/httpfs/include/httpfs.h

```diff
@@ -30,7 +45,9 @@ struct HTTPParams {
 struct HTTPFileInfo : public common::FileInfo {
     HTTPFileInfo(std::string path, common::FileSystem* fileSystem, int flags);
 
-    void initializeClient();
+    virtual void initialize();
+
+    virtual void initializeClient();
```

Comment:
Double check if this function needs to be public.

Reply:
I think it is a struct and we never write visibility specifier for struct